Bottom: d7fffdc40073d95d98b9d5c01c5c865b7caa0c69
Top:    cd61bfbc2c99b1f91308eee9fbeff14699c5ada4
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-03-05 17:19:16 -0800

Refresh of release

---

diff --git a/librdmacm.spec.in b/librdmacm.spec.in
index 575ce84..7ad6c0c 100644
--- a/librdmacm.spec.in
+++ b/librdmacm.spec.in
@@ -1,7 +1,7 @@
 %define ver @VERSION@
 
 Name: librdmacm
-Version: 1.0.16
+Version: 1.0.17
 Release: 1%{?dist}
 Summary: Userspace RDMA Connection Manager
